libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -161,6 +161,14 @@ export interface AccountBridge<T extends TransactionCommon> {
     parentAccount?: Account | null | undefined;
     transaction?: T | null | undefined;
   }): Promise<BigNumber>;
+  applyReconciliation?: (
```

Comment:
these 4 added functions must be documented in a comment.

Reply:
Fixed. Thanks for the feedback.